src/styles/blocks/about.scss

```diff
@@ -0,0 +1,52 @@
+.about {
+  &__title {
+    grid-column: 1 / 3;
+
```

Comment:
general check all scss files and remove spaces

```
 grid-column: 1 / 3;
 margin-bottom: 32px;
 font-weight: 700;

```
add spaces only between blocks, example:
```
 &__description {
    grid-column: 1 / 3;

    @include onTablet {
      grid-column: 4 / 7;
    }
```